chat-plugins/info.js

```diff
@@ -1808,7 +1808,7 @@ exports.commands = {
 		let offset = 0;
 		let removeOutlier = 0;
 
-		let modifierData = target.match(/[\-\+]/);
+		let modifierData = target.match(/[-+]/);
```

Comment:
In general, I prefer `[` blocks in regexes that end with `-` to ones that start with `-`, and we should enforce that in code.